server.js

```diff
@@ -1,10 +1,54 @@
 const express = require("express");
 const serveStatic = require("serve-static");
+const multer = require("multer");
+const crypto = require("crypto");
+const AWS = require("aws-sdk");
+const cors = require("cors");
 
 module.exports = function() {
   const port = process.env.C20_PORT ? Number(process.env.C20_PORT) : 8080;
+  const bucketName = process.env.C20_S3_BUCKET || "reclaimers-data-test";
+  const s3 = new AWS.S3();
```

Comment:
Just a note on why there's no API key configured here. The AWS SDK will first try the standard `AWS_*` environment variables (which are mainly used when running locally for testing) but will ultimately fall back on the "role" of the EC2 instance (or other service) it runs on. Outside of this PR, I have provisioned an EC2 instance with this branch deployed to it and running, which is what you hit when you go to http://api.reclaimers.net:8080/. That instance's role (and therefore any use of the AWS SDK on that host) is given permission to write to a specific S3 bucket and object, which I configured to be publicly readable at: https://reclaimers-data-test.s3.amazonaws.com/survey-results.json.